src/util/transition/ngbFadingTransition.ts

```diff
@@ -3,3 +3,21 @@ import {NgbTransitionStartFn} from './ngbTransition';
 export const ngbAlertFadingTransition: NgbTransitionStartFn = ({classList}: HTMLElement) => {
   classList.remove('show');
 };
+
+export interface NgbToastCtx { direction: 'show' | 'hide'; }
+
+export const ngbToastFadeInTransition: NgbTransitionStartFn<NgbToastCtx> = ({classList}: HTMLElement) => {
+  classList.remove('hide');
+  classList.add('showing');
+
+  return () => {
+    classList.remove('showing');
+    classList.add('show');
+  };
+};
+
+export const ngbToastFadeOutTransition: NgbTransitionStartFn<NgbToastCtx> = ({classList}: HTMLElement) => {
```

Comment:
Don't forget to remove the context in the types also

Reply:
Sure, thanks !